common/src/main/java/com/sequenceiq/cloudbreak/ccm/cloudinit/DefaultCcmParameters.java

```diff
@@ -0,0 +1,56 @@
+package com.sequenceiq.cloudbreak.ccm.cloudinit;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Default {@link CcmParameters} implementation.
+ */
+public class DefaultCcmParameters implements CcmParameters, Serializable {
+
+    private static final long serialVersionUID = 1L;
```

Comment:
Do we rely on the Java default serialization format? If so, having the same `serialVersionUID` for all classes below might not be the best practice. 😃

Reply:
The specific value of the serialVersionUID does not matter, and does not need to be globally unique. It only needs to be unique within a single class, and only needs to change values when the class is modified in a way that is not serialization compatible. Setting the initial value to 1L is one convention. It happens to be the one we use in Director, but it is also used elsewhere in the CB codebase, such as in SecretProxy.

Reply:
Alright, let us keep following that convention then. 👍